pkg/helpers/pki_test.go

```diff
@@ -103,10 +102,59 @@ func TestSubjectAltNameInCert(t *testing.T) {
 
 	// Test SAN is not empty
 	SubjectAltNames := []string{"santest.mydomain.com", "santest2.testdomain.net"}
-	masterExtraFQDNs := append(formatAzureProdFQDNs(dnsPrefix), SubjectAltNames...)
+	formattedDNSPrefixes := []string{
```

Comment:
It's too bad this data structure is a copy of the one in pkg/acsengine/engine_test.go, but since they're in different packages and it's just test code, I guess that's ok.